shared/src/main/scala-2.12/zipper/ForImpl.scala

```diff
@@ -7,6 +7,16 @@ import scala.collection.generic.CanBuildFrom
 import scala.language.higherKinds
 
 private[zipper] trait ForImpl {
+  implicit def `Unzip List-based`[A, L <: HList](
+    implicit generic: Generic.Aux[A, L],
+    select: Selector[L, List[A]],
+    replace: Replacer.Aux[L, List[A], List[A], (List[A], L)]
+  ): Unzip[A] = new Unzip[A] {
+    def unzip(node: A): List[A] = select(generic.to(node))
+
+    def zip(node: A, children: List[A]): A = generic.from(replace(generic.to(node), children)._2)
+  }
```

Comment:
Oh, it seems like we could put this into the common Scala 2 source file. Let me try to patch it.